syncd/Syncd.cpp

```diff
@@ -1,3 +1,4 @@
+#include "config.h"
```

Comment:
@Junchao-Mellanox why do we need this?

Reply:
The macro `#define MELLNOX 1` is generated to config.h.